<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WSetting_GeneralSettings</class>
 <widget class="QWidget" name="WSetting_GeneralSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1370</width>
    <height>752</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1370</width>
     <height>750</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: none;
border-image: none;
background-image: none;

background: #FFFFFF;
border-radius: 20px;</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>270</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>100</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>PingFang SC</family>
      <pointsize>-1</pointsize>
      <weight>62</weight>
      <bold>true</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 43px;
height: 22px;
font-family: PingFang SC;
font-weight: 500;
font-size: 30px;
color: #333333;
line-height: 35px;</string>
    </property>
    <property name="text">
     <string>试剂孔位随量跟踪</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_level_tracking">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>104</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1030</width>
      <height>50000</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/beijing/emergency_none.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="auto_mix">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>180</y>
      <width>104</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1030</width>
      <height>50000</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/beijing/emergency_none.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="auto_mix_label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>180</y>
      <width>270</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>100</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>PingFang SC</family>
      <pointsize>-1</pointsize>
      <weight>62</weight>
      <bold>true</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 43px;
height: 22px;
font-family: PingFang SC;
font-weight: 500;
font-size: 30px;
color: #333333;
line-height: 35px;</string>
    </property>
    <property name="text">
     <string>自动混匀使能</string>
    </property>
   </widget>
   <widget class="QPushButton" name="auto_load">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>300</y>
      <width>104</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1030</width>
      <height>50000</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/beijing/emergency_none.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="auto_load_label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>300</y>
      <width>270</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>100</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>PingFang SC</family>
      <pointsize>-1</pointsize>
      <weight>62</weight>
      <bold>true</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 43px;
height: 22px;
font-family: PingFang SC;
font-weight: 500;
font-size: 30px;
color: #333333;
line-height: 35px;</string>
    </property>
    <property name="text">
     <string>自动上料使能</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>505</x>
      <y>650</y>
      <width>360</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>设置需要重启生效</string>
    </property>
   </widget>
   <widget class="QLabel" name="auto_fill_bottle_label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>420</y>
      <width>301</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>100</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>PingFang SC</family>
      <pointsize>-1</pointsize>
      <weight>62</weight>
      <bold>true</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 43px;
height: 22px;
font-family: PingFang SC;
font-weight: 500;
font-size: 30px;
color: #333333;
line-height: 35px;</string>
    </property>
    <property name="text">
     <string>自动灌注缓冲瓶使能</string>
    </property>
   </widget>
   <widget class="QPushButton" name="auto_fill_bottle">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>420</y>
      <width>104</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1030</width>
      <height>50000</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/beijing/emergency_none.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
